packages/terra-hookshot/tests/nightwatch/HookshotTemplate.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Hookshot from '../../lib/Hookshot';
+import HookshotContent from './HookshotContentTemplate';
+
+const propTypes = {
+  /**
+   * The hookshot contents props. Defaults to closesOnEsc=true, closeOnOutsideClick=true & closeOnResize=true.
+   * onRequestClose is provided by the template.
+   */
+  hookshotContentProps: PropTypes.any,
+  id: PropTypes.string,
+  isOpen: PropTypes.bool,
+  contentAttachment: PropTypes.string,
+  targetAttachment: PropTypes.string,
+  buttonMarginLeft: PropTypes.string,
+  buttonMarginRight: PropTypes.string,
+  buttonMarginTop: PropTypes.string,
+  boundingWidth: PropTypes.string,
+  attachmentMargin: PropTypes.number,
+};
+
+const defaultProps = {
+  hookshotContentProps: { },
+  id: 'hookshot',
+  isOpen: false,
+  contentAttachment: 'middle right',
+  targetAttachment: 'middle left',
+  buttonMarginTop: '55px',
+  buttonMarginLeft: '220px',
+  buttonMarginRight: '220px',
+  boundingWidth: '540px',
+};
+
+class HookshotTemplate extends React.Component {
```

Comment:
Picky, but should this be renamed to TestingTemplate? HookshotTemplate makes it seems like a component we can extract out of this folder and move to the src folder.